### DISCUSSION TITLE
Cannot undo 'Filter Table' in dbeaver 3.7.0

### DISCUSSION THREAD

**Discussion by MehrnazSh** [dbeaver/dbeaver#566] [Q&A] [answered]
I used the feature 'Filter Table' but cannot undo it. Is the table dropped? I expected this feature to hide the table until I want it back. Otherwise I could have easily dropped it.

**Comment by serge-rider** [comment, marked as answer]
No, table was just hidden.
You can bring it back by clicking on "Filter tables ..." action on "Tables" node context menu.

**Comment by MehrnazSh** [comment]
Thanks for your quick response.
